server/src/main/java/org/apache/iotdb/db/engine/StorageEngineV2.java

```diff
@@ -555,6 +560,62 @@ public void forceCloseAllProcessor() throws TsFileProcessorException {
     }
   }
 
+  public void closeStorageGroupProcessor(String storageGroupPath, boolean isSeq, boolean isSync) {
+    for (DataRegion dataRegion : dataRegionMap.values()) {
+      if (dataRegion.getLogicalStorageGroupName().equals(storageGroupPath)) {
+        for (TsFileProcessor tsFileProcessor : dataRegion.getWorkSequenceTsFileProcessors()) {
+          dataRegion.syncCloseOneTsFileProcessor(isSeq, tsFileProcessor);
+        }
+      }
+    }
+  }
+
+  public TSStatus operatorFlush(TFlushReq req) throws StorageGroupNotSetException {
+    if (req.storageGroups == null) {
+      StorageEngineV2.getInstance().syncCloseAllProcessor();
+      WALManager.getInstance().deleteOutdatedWALFiles();
+    } else {
+      for (String storageGroup : req.storageGroups) {
+        if (req.isSeq == null) {
+          StorageEngineV2.getInstance().closeStorageGroupProcessor(storageGroup, true, req.isSync);
```

Comment:
after remove the isSync clause, the default is true

Reply:
fixed~